libs/blocks/gnav/gnav-appLauncher.js

```diff
@@ -0,0 +1,50 @@
+import { createTag, makeRelative } from '../../utils/utils.js';
```

Comment:
This is just a general comment - I couldn't see the More option anymore. Is this not being done ? 
![Screenshot 2022-11-10 at 12 08 04](https://user-images.githubusercontent.com/115231412/201076098-54dc59f7-30b1-46d5-a989-60188eea36e6.png)

Reply:
I had a chat with the Consonant Design team (John Spencer). The thought was this feature is rarely used. So, I opted to leave it out for simplicity sake and plan to add if anyone yells loud enough.

Reply:
ok, perfect !